testing/tslib/APKBUILD

```diff
@@ -0,0 +1,30 @@
+# Contributor: Clayton Craft <clayton@craftyguy.net>
+# Maintainer: Clayton Craft <clayton@craftyguy.net>
+pkgname=tslib
+pkgver=1.13
+pkgrel=0
+pkgdesc="Touchscreen Access Library"
+arch="all"
+url="https://github.com/kergoth/tslib"
+license="GPL2"
```

Comment:
According to the [source](https://github.com/kergoth/tslib/blob/master/COPYING) is the license "GNU Lesser General Public License v2.1", thus `LGPL2+`.